samples/snippets/csharp/VS_Snippets_CLR_System/System.AttributeUsageAttribute.Inherited/cs/Inherited1.cs

```diff
@@ -42,19 +42,19 @@ public static void Main()
                         memberA.GetCustomAttributes(typeof(InheritedAttribute), true).Length > 0); 
       
       Type typeB = typeof(DerivedB);
-      Console.WriteLine("DerivedB has Inherited attribute: {0}", 
-                        typeB.GetCustomAttributes(typeof(InheritedAttribute), true).Length > 0); 
+      Console.WriteLine("DerivedB has NotInherited attribute: {0}", 
+                        typeB.GetCustomAttributes(typeof(NotInheritedAttribute), true).Length > 0); 
       MethodInfo memberB = typeB.GetMethod("MethodB");
```

Comment:
Would be nice to use `nameof` operator here: `MethodInfo memberB = typeB.GetMethod(nameof(DerivedB.MethodB));`

In line 40, I suggest the same change to `MethodInfo memberA = typeA.GetMethod("MethodA");`